### PR TITLE
Fix/mongodb var

### PR DESCRIPTION
**Pull request by conkerant**
The variable MONGO_PORT env var need to be more explicit and automated.

### CODE REVIEW

**Comment by nireeshT** [comment]
@conkerant  Please update branch

File: api-audit/docker/api-audit-properties-builder.sh [hunk 1]
```diff
@@ -5,12 +5,8 @@ if [ "$SKIP_PROPERTIES_BUILDER" = true ]; then
   exit 0
 fi
 
-# if we are linked, use that info
-if [ "$MONGO_PORT" != "" ]; then
-  # Sample: MONGO_PORT=tcp://172.17.0.20:27017
-  export SPRING_DATA_MONGODB_HOST=`echo $MONGO_PORT|sed 's;.*://\([^:]*\):\(.*\);\1;'`
-  export SPRING_DATA_MONGODB_PORT=`echo $MONGO_PORT|sed 's;.*://\([^:]*\):\(.*\);\2;'`
-fi
+SPRING_DATA_MONGODB_HOST=${MONGODB_HOST:-db}
```
Comment (matthewbeasley) [inline api-audit/docker/api-audit-properties-builder.sh]: This change is correct and desired but I believe it should be done lower down,

**Comment by matthewbeasley** [comment]
@conkerant could you update your branch then makes the requested changes? This looks good and we want to merge it.

File: api-audit/docker/api-audit-properties-builder.sh (hunk 1 above)
Comment (conkerant) [reply api-audit/docker/api-audit-properties-builder.sh]: Hi,
At line 8 it's the db host, at line 17 it's db name.

**Comment by conkerant** [comment]
All done

**Comment by conkerant** [comment]
Hi, the repo has been rebased from latest commits

**Pull request closed without merging**